ibm_catalog.json

```diff
@@ -400,27 +401,25 @@
             }
           ],
           "architecture": {
-            "descriptions": "This architecture supports the deployment of Log Analysis and Cloud Monitoring instances on IBM Cloud. It also supports configuring the Activity Tracker event routing.",
+
+            "descriptions": "This architecture supports the deployment of IBM Log Analysis and IBM Cloud Monitoring instances on IBM Cloud. It also supports the deployment of IBM Cloud Activity Tracker Event Routing to either an Object Storage bucket or Log Analysis target.",
+
             "features": [
               {
-                "title": "Creates the Log Analysis instance.",
-                "description": "Creates and configures the Log Analysis instance."
+                "title": "Creates a Log Analysis instance",
+                "description": "Creates and configures the IBM Cloud Log Analysis instance."
               },
               {
-                "description": "Creates the Cloud Monitoring instance.",
-                "title": "Creates and configures the Cloud Monitoring instance."
+                "description": "Creates a Monitoring instance",
+                "title": "Creates and configures the IBM Cloud Monitoring instance."
               },
               {
-                "title": "Creates the KMS encrypted COS bucket required for the logs archiving.",
-                "description": "Creates and configures the COS bucket with archiving and expiration enabled."
+                "title": "Creates KMS-encrypted Object Storage buckets for log files and Activity Tracker events",
+                "description": "Creates and configures the IBM Cloud Object Storage buckets with archiving and expiration enabled."
               },
               {
-                "title": "Creates the KMS encrypted COS bucket required for routing of ATracker events.",
-                "description": "Creates and configures the COS bucket with archiving and expiration enabled."
-              },
-              {
-                "title": "Support Configuring the Activity Tracker event routing.",
-                "description": "Supports configuring the Activity Tracker event routing to a COS bucket and to a Log Analysis instance."
+                "title": "Creates an IBM Cloud Activity Tracker event route and Object Storage and Log Analysis targets",
```

Comment:
```suggestion
                "title": "Creates an IBM Cloud Activity Tracker event route to an Object Storage and Log Analysis targets",
```

Reply:
assumed that `bucket` was missing from COS, so added that.
If that's not correct, then the existing should be ok.

Reply:
adding bucket and making target singular works perfectly fine.